src/components/changePassword/changePassword2.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
I don't understand why you created the separate test file **changePassword2.test.js** instead of just adding the test to the file **changePassword.test.js**